waggle-dance-api/src/main/java/com/hotels/bdp/waggledance/api/WaggleDanceException.java

```diff
@@ -15,15 +15,10 @@
  */
 package com.hotels.bdp.waggledance.api;
 
+import lombok.experimental.StandardException;
+
+@StandardException
```

Comment:
Could you please revert these, I'm having issues in my environment compiling this, not sure why the other annotations are find and we're using lombok internally in other projects. But since this is an experimental features maybe it's best we don't rely on that.

Reply:
Reverted